### PR TITLE
cilium: prep work for nat46/64 k8s integration

### PR DESCRIPTION
**Pull request by borkmann**
(See commit desc)

### CODE REVIEW

File: Documentation/network/clustermesh/affinity.rst [hunk 1]
```diff
@@ -32,15 +32,15 @@ remote endpoints if and only if all of local backends are not available or unhea
    metadata:
      name: rebel-base
      annotations:
-        io.cilium/global-service: "true"
+        io.cilium.service.global: "true"
```
Comment (sayboras) [inline Documentation/network/clustermesh/affinity.rst:35]: if we consider io.cilium.service as prefix and global is the name, should the annotation be io.cilium.service/global instead ? :thinking: 

https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set

**Review by sayboras — COMMENTED** [review]
Only one comment as per below, keen to hear from others who know more about convention in k8s as well.

**Review by aspsk — COMMENTED** [review]
Looks good. I agree with @sayboras that the names should be separated with the slash and we need to leave the `io.cilium.service` as a prefix

**Review by qmonnet — APPROVED** [review]
Looks good to me

File: Documentation/network/clustermesh/affinity.rst (hunk 1 above)
Comment (borkmann) [reply Documentation/network/clustermesh/affinity.rst:35]: It should actually be `service.cilium.io/global: "true"`

See also `kubernetes.io/egress-bandwidth` example in https://docs.cilium.io/en/stable/gettingstarted/bandwidth-manager/ .
Will fix.

Comment (borkmann) [reply Documentation/network/clustermesh/affinity.rst:35]: oh well, looks all of them are messed up.. :/

Comment (aanm) [reply Documentation/network/clustermesh/affinity.rst:35]: @borkmann if we are changing this then we should fix and make them all consistent as well as add a note in the upgrade docs mentioning that we are changing this label.

**Review by aanm — CHANGES_REQUESTED** [review]

**Pull request closed without merging**